pkg/crd/known_types.go

```diff
@@ -84,6 +85,13 @@ var KnownPackages = map[string]PackageOverride{
 		}
 		// No point in calling AddPackage, this is the sole inhabitant
 	},
+
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1": func(p *Parser, pkg *loader.Package) {
+		p.Schemata[TypeIdent{Name: "JSON", Package: pkg}] = apiext.JSONSchemaProps{
+			XPreserveUnknownFields: pointer.BoolPtr(true),
```

Comment:
can you just inline this?  I see very little reason to pull in a library for this

Reply:
done